nrf_802154/doc/CHANGELOG.rst

```diff
@@ -9,6 +9,16 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+Main branch
+***********
+
+Bug fixes
+=========
+
+* For delayed transmission on nRF5340 when passing NULL metadata the channel now defaults to the value in PIB.
+  It used to be 11, which was inconsistent with the behaviour on nRF52. (KRKNWK-13539)
```

Comment:
```suggestion
* Fixed an issue where the channel for the delayed transmission on the nRF5340 SoC when passing NULL metadata would be set to 11.
  This was inconsistent with the behavior on nRF52 and the channel now defaults to the value in PIB. (KRKNWK-13539)
```
Two more edits needed:

- nRF52 -- we can't use just "nRF52"; use either the full number of the SoC or "nRF52 Series' SoCs".
- Expand PIB to full name with "(PIB)" at the end.

Also, we use American English, so "behavior", not "behaviour".

Reply:
Fixed